### PR TITLE
Use Admin Builder instead of Pages Builder

### PR DESCRIPTION
**Pull request by GustavoMA**
Branch used to test out vtex/builder-hub#478, vtex/admin-navigation#1

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -11,12 +11,9 @@
   "dependencies": {
     "vtex.styleguide": "9.x"
   },
-  "peerDependencies": {
-    "vtex.admin": "2.x"
-  },
   "builders": {
     "react": "3.x",
-    "pages": "1.x",
+    "admin": "0.x",
```
Comment (kevinch) [inline manifest.json:16]: Shouldn't it be `2.x`?

File: react/AdminExample.tsx [hunk 2]
```diff
@@ -1,12 +1,25 @@
...
+import React, { Component } from "react";
+import { FormattedMessage } from "react-intl";
+import { Layout, PageBlock, PageHeader } from "vtex.styleguide";
 
 class AdminExample extends Component {
   public render() {
     return (
-      <FormattedMessage id="admin-example.hello-world" />
-    )
+      <div className="h-100">
```
Comment (kevinch) [inline react/AdminExample.tsx]: Should be useless with Styleguide `8.x`: https://kevin--ambienteqa.myvtex.com/admin/customercredit/config `<Layout` is directly used: https://github.com/vtex/admin-customer-credit/blob/feature/state-query-view-pattern/react/pages/configuration/view.js#L51

File: manifest.json (hunk 1 above)
Comment (ohmyguigs) [reply manifest.json:16]: nope, it's the builder, not the admin app 😛

File: react/AdminExample.tsx (hunk 2 above)
Comment (ohmyguigs) [reply react/AdminExample.tsx]: this doesn't even work 😢

Comment (ohmyguigs) [reply react/AdminExample.tsx]: I made a fix in the next commit, but this should not be the final solution :'(

**Review by kevinch — APPROVED** [review]

**Pull request merged**